x/ecocredit/params_test.go

```diff
@@ -0,0 +1,44 @@
+package ecocredit
+
+import "testing"
+
+func Test_validateCreditTypes(t *testing.T) {
+	tests := []struct {
```

Comment:
Maybe add one with the wrong type?

Reply:
this test just tests against the validation function which only really catches a few things: 

1. duplicates 
2. empty credit type names
3. empty credit units
4. duplicates

do we have a canonical list of what could be valid types that im missing here?

Reply:
I meant something that's not a `*[]CreditType` like we have in other validation tests, because we might pass in e.g. a `bool`!

Reply:
And now you might want to add a test that says an non-normalized name fails